scripts/quests/ahtUrhgan/Promotion_Lance_Corporal.lua

```diff
@@ -0,0 +1,252 @@
+-----------------------------------
+-- Promotion: Lance Corporal
+-- Log ID: 6, Quest ID: 92
+-- Naja Salaheem !pos 26 -8 -45.5 50
+-----------------------------------
+local ID = require("scripts/zones/Aht_Urhgan_Whitegate/IDs")
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/npc_util')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.PROMOTION_LANCE_CORPORAL)
+
+local testTubes =
+{
+    [16986736] = { xi.ki.EMPTY_TEST_TUBE_1, xi.ki.TEST_TUBE_1 },
+    [16986737] = { xi.ki.EMPTY_TEST_TUBE_2, xi.ki.TEST_TUBE_2 },
+    [16986738] = { xi.ki.EMPTY_TEST_TUBE_3, xi.ki.TEST_TUBE_3 },
+    [16986739] = { xi.ki.EMPTY_TEST_TUBE_4, xi.ki.TEST_TUBE_4 },
+}
+
+quest.reward =
+{
+    keyItem = xi.ki.LC_WILDCAT_BADGE,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getQuestStatus(xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.PROMOTION_SUPERIOR_PRIVATE) == QUEST_COMPLETED and
+                player:getCharVar("AssaultPromotion") >= 25
```

Comment:
This should really be == 25 as they should not be able to accrue more than 25..

Reply:
I don't believe there's a check preventing them from reaching more than 25. It just keeps adding and adding

Reply:
Oh ok - but when you do the promotion quest it sets you to 0 .. i see